### PR TITLE
Hyperboria DNS, Time servers and Clock Handling

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Added installation NTP and application of hyperboria compatible time server
Added installation of fake-hwclock and save the time every 5 mins instead of every hour

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Merged in Time PR to make life easier :)

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ The following instructions will help you set up an encrypted mesh network on Ras
 | `WITH_PROMETHEUS_SERVER`        | **9090**: Prometheus Server UI                 | Set to `true` if you want to install [Prometheus Server](https://github.com/prometheus/prometheus) to collect network metrics. *Requires Prometheus Node Exporter.* |
 | `WITH_GRAFANA`                  | **3000**: Grafana UI (login: admin/admin)      | Set to `true` if you want to install [Grafana](https://grafana.com) to display network metrics. *Requires Prometheus Server.* |
 | `WITH_EXTRA_TOOLS`              | None                                           | Set to `true` if you want to install non-essential tools useful for network analysis. |
+| `WITH_H_DNS`                    | None                                           | Set to `true` if you want ot replace your DNS servers with Hyperboria compatible ones. |
```
Comment (benhylau) [inline README.md]: `to`

File: README.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ The following instructions will help you set up an encrypted mesh network on Ras
...
+| `WITH_H_NTP`                    | None                                           | Set to `true` if you want to install Hyperbora friendly time servers. |
```
Comment (benhylau) [inline README.md]: `Hyperboria`

File: scripts/install2 [hunk 3]
```diff
@@ -213,6 +250,30 @@ if [[ $BOARD_REVISION == *"a02082"* || $BOARD_REVISION == *"a22082"* ]] && [ ! -
     source hostapd/install
 fi
 
+# DNS Server Settings
```
Comment (benhylau) [inline scripts/install2]: We should create folders with `install` and `uninstall` scripts. This file only runs those install scripts.

**Pull request merged**